src/utils/__tests__/actions.spec.js

```diff
@@ -576,7 +576,7 @@ describe("actions", () => {
         isEnabled: jest.fn().mockReturnValue(true),
       };
       const result = getActionsList(shouldHide);
-      expect(result.length).toBe(20);
+      expect(result.length).toBe(21);
```

Comment:
What does unsupported feature navigations means? Is cyclic group an unsupported feature navigation?

Reply:
This is for apps running in Direct Query mode. Unsupported features are controlled by engine and stored on the app layout. It would be good to verify if cyclic groups is something that can or cannot work in DQ apps

Reply:
Ok. Björn have noted this in our test plan. I guess in the mean time we let this test be like this until we know it has support or not